ngrinder-controller/src/main/java/org/ngrinder/script/service/GitHubFileEntryService.java

```diff
@@ -278,25 +278,26 @@ public Set<GitHubConfig> getAllGitHubConfig(User user) throws FileNotFoundExcept
 		return getAllGithubConfig(gitConfigYaml);
 	}
 
-	private Set<GitHubConfig> getAllGithubConfig(FileEntry gitConfigYaml) {
+	protected Set<GitHubConfig> getAllGithubConfig(FileEntry gitConfigYaml) {
```

Comment:
Why modifier was changed?

Reply:
I was changed it for adding a unit test. But, it doesn't need to change access modifier.
I will change it back to private and add a unit test to verify that successfully parse the `.gitconfig.yml` file.